src/lib/factories/dealer.ts

```diff
@@ -0,0 +1,18 @@
+import { SearchDealer } from "../../types/models/dealer"
+
+export function DealerPromotion(): SearchDealer {
+  return {
+    id: 0,
+    location: {
+      address: "string",
```

Comment:
Dont you wanna return some dummy data or the property instead of string for every property?